src/test/scala/com/snowplowanalytics/iglu/ctl/commands/ITHelpers.scala

```diff
@@ -0,0 +1,67 @@
+package com.snowplowanalytics.iglu.ctl.commands
+
+import cats.effect._
+import cats.effect.implicits._
+import com.dimafeng.testcontainers.MockServerContainer
+import org.mockserver.client.MockServerClient
+import org.mockserver.mock.Expectation
+import org.mockserver.model.HttpRequest.request
+import org.mockserver.model.HttpResponse.response
+import org.testcontainers.lifecycle.Startable
+
+import scala.concurrent.ExecutionContext
+
+object ITHelpers {
+
+  val executionContext: ExecutionContext = ExecutionContext.global
+  implicit val ioContextShift: ContextShift[IO] = IO.contextShift(executionContext)
+  implicit val ioTimer: Timer[IO] = IO.timer(executionContext)
+
+  def testSchema(fields: String): String =
+    s"""
+       |{
+       |  "$$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+       |  "description": "Test schema",
+       |  "self": {
+       |    "vendor": "com.test",
+       |    "name": "test",
+       |    "format": "jsonschema",
+       |    "version": "1-0-0"
+       |  },
+       |  "type": "object",
+       |  "properties": $fields
+       |}
+       |""".stripMargin
+
+  def testSchemaWrongType(fields: String, minor: Int): String =
```

Comment:
I know you moved it from `TableCheckITSpec`, but what do think about making it slightly more readable?

I'm thinking about leaving only one function `testSchema` which would accept 2 args: fields and version as a string. Sth like this:

```
  def testSchema(fields: String, version: String): String =
    s"""
       |{
       |  "$$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
       |  "description": "Test schema",
       |  "self": {
       |    "vendor": "com.test",
       |    "name": "test",
       |    "format": "jsonschema",
       |    "version": "$version"
       |  },
       |  "type": "object",
       |  "properties": $fields
       |}
       |""".stripMargin
```

Then in tests we could be more explicit what version of test schema we use:

```
...
testSchema(fields =..., version = "1-0-0"),
testSchema(fields =...., version = "1-0-1")
...
```

WDYT?

Reply:
👍 good point